src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+
+    Downloads all inputs to the sub job, including:
+        - interval lists
+        - basecount files
+        - ploidy files
+    '''
+    mark-section "Downloading files to sub job"
+
+    SECONDS=0
+    dx-download-all-inputs --parallel
+
+    interval_list=$( basename $( find /home/dnanexus/in/ -name '*.interval_list' ))
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)